packages/rolldown/src/config.ts

```diff
@@ -0,0 +1,8 @@
+import { version } from '../package.json';
+
+import { defineConfig } from './utils/define-config';
+import { loadConfig } from './utils/load-config';
+
+export { defineConfig, loadConfig };
+
+export const VERSION: string = version;
```

Comment:
The packages/rolldown/src/index.ts already export `VERSION `

Reply:
I added it to this file since it's intended as a separate entrypoint (`rolldown/config`) to not have to import the entire `rolldown`, but like I said I'm not sure it would be helpful since `loadConfig` imports the compiler anyways

Reply:
@underfin browser build includes `index.ts` entry, and it cannot include fs API (`loadConfig`) from `config.ts`.